sidecar-injection/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+)
+
+func main() {
+	var parameters WhSvrParameters
+
+	// get command line parameters
+	flag.IntVar(&parameters.port, "port", 443, "Webhook server port.")
+	flag.StringVar(&parameters.certFile, "tlsCertFile", "/etc/webhook/certs/cert.pem", "File containing the x509 Certificate for HTTPS.")
```

Comment:
I think we should follow the convention used by the other controllers' flags i.e. `tls-cert-file`, instead of `tlsCertFile`.